<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookmarkWidget</class>
 <widget class="QWidget" name="BookmarkWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>192</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>&amp;Name</string>
     </property>
     <property name="buddy">
      <cstring>nameLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="nameLineEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="timeoutLabel">
     <property name="text">
      <string>&amp;Timeout</string>
     </property>
     <property name="buddy">
      <cstring>timeoutComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="timeoutComboBox"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="tagsLabel">
     <property name="text">
      <string>T&amp;ags</string>
     </property>
     <property name="buddy">
      <cstring>tagsLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="descriptionLabel">
     <property name="text">
      <string>&amp;Description</string>
     </property>
     <property name="buddy">
      <cstring>descriptionTextEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPlainTextEdit" name="descriptionTextEdit">
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QWidget" name="tagsWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="tagsLineEdit">
        <property name="placeholderText">
         <string>List of tags divided by commas or spaces</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="tagsListLabel">
        <property name="text">
         <string/>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>descriptionTextEdit</tabstop>
  <tabstop>timeoutComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
